sites/next.skeleton.dev/src/examples/components/tabs/VerticalIconsExample.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { Tabs, TabsList, TabsControl } from '@skeletonlabs/skeleton-react';
+import { LibraryBig, Film, Tv } from 'lucide-react';
+
+export const Page: React.FC<any> = () => {
+	const [group, setGroup] = useState('books');
+	return (
+		<Tabs>
+			<Tabs.List>
+				<TabsList.Control name="books" group={group} onchange={setGroup} title="Books">
+					<TabsControl.Item classes="flex-col">
```

Comment:
Each `TableControl.Item` needs to use`rounded="rounded-container"` to properly handle the hover state. The default `rounded` class is much too rounded.

Before:

![image](https://github.com/skeletonlabs/skeleton/assets/1509726/27ab954d-5655-4cab-b413-e9148f0731c3)

After:

![Screenshot 2024-04-09 at 2 26 10 PM](https://github.com/skeletonlabs/skeleton/assets/1509726/1debc343-e933-41e3-afec-c0a26f950b05)

Reply:
sorry it looks like I have changed it only for svelte on your initial request, I will include it for react too.